miio/discovery.py

```diff
@@ -174,6 +183,13 @@
     "chunmi-cooker-normal3": Cooker,
     "chunmi-cooker-normal4": Cooker,
     "chunmi-cooker-normal5": Cooker,
+    MODEL_EXP1: IHCooker,
+    MODEL_FW: IHCooker,
+    MODEL_TW1: IHCooker,
+    MODEL_KOREA1: IHCooker,
+    MODEL_HK1: IHCooker,
+    MODEL_V1: IHCooker,
+    MODEL_EG1: IHCooker,
```

Comment:
I don't think these are correct for mdns? Either remove these completely for now, or check what format is being used in mdns on your device and just add them here as regular strings.

This information should be moved inside the integration itself at some point, but it'll require some more work. Having these here as strings makes it easy to copy them over if/when that time comes.

Reply:
Removed.

Reply:
They seem to be still there in the diff :eyes: